src/ewatercycle/_forcings/caravan.py

```diff
@@ -209,6 +209,16 @@ def generate(  # type: ignore[override]
 
         for temp in ["tas", "tasmin", "tasmax"]:
             ds_basin_time[temp].attrs.update({"height": "2m"})
+            #convert units to Kelvin for compatiabillity with NetCDF-CF conventions
+            if (ds_basin[temp].attrs["unit"]) == "°C":
+                ds_basin[temp].values = ds_basin[temp].values + 273.15
+                ds_basin[temp].attrs["unit"] = "°K"
+        
+        for var in ["evspsblpot", "pr"]:
+            #convert units to kg m-2 s-1 for compatiabillity with NetCDF-CF conventions
+            if (ds_basin[var].attrs["unit"]) == "mm":
+                ds_basin[var].values = ds_basin[temp].values / (86400) #NOTE THAT THIS CONVERSION ASSUMES DAILY DATA
```

Comment:
"All time series data except streamflow are aggregated (daily and spatially over basins)", Kratzert 2023.

And all these other time series data is merged with the streamflow on the same time dimension in the netCDF files we host on 4TU. So I think it's safe to hard code the conversion here.
```suggestion
                ds_basin[var].values = ds_basin[temp].values / (86400)  # mm/day --> kg m-2 s-1
```